components/region_cache_memory_engine/src/background.rs

```diff
@@ -82,6 +89,8 @@ impl Display for GcTask {
     }
 }
 
+pub(crate) type LoadAndPrepareRangeFn =
```

Comment:
Please add some comments for this as well as `PdRangeHintService`.

Reply:
Done.